src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpExtractMethod.cs

```diff
@@ -114,6 +114,7 @@ public int Method()
 
             MarkupTestFile.GetSpans(expectedMarkup, out var expectedText, out ImmutableArray<TextSpan> spans);
             Assert.Equal(expectedText, VisualStudio.Editor.GetText());
+            VisualStudio.Workspace.WaitForAsyncOperations(Helper.HangMitigatingTimeout, FeatureAttribute.Rename);
```

Comment:
Is it possible to put this inside GetTagSpans so all callers don't have to remember?

Reply:
Updated. Can you let me know if the second commit is an improvement?